core/src/main/scala/chisel3/experimental/package.scala

```diff
@@ -130,6 +132,32 @@ package object experimental {
     }
   }
 
+  /** This class provides the `Lit` method needed to define a `Vec` literal
+    */
+  object VecLiterals {
+    implicit class AddVecLiteralConstructor[T <: Vec[_]](x: T) {
+      /** Given a generator of a list tuples of the form [Int, Data]
+        * constructs a Vec literal, parallel concept to `BundleLiteral`
+        *
+        * @param elems generates a lit of `(Int, Data)` where the I
```

Comment:
I think the sentence was accidentally

Reply:
Good catch, deleted dangling string from comment

Reply:
Is it accurate to describe the `elems` argument as "generates a lot" as opposed to something more like the values of the lit?

Reply:
done